### PR TITLE
chore: HTML cleanup

### PR DESCRIPTION
**Pull request by Rupurudu**
I did bunch of HTML code cleanup on the repo.

Check commit messages for details.

- [x] This PR is not from my *main* or *master* branch :poop:, but a *separate* branch :white_check_mark:
- [x] Each commit has a valid :black_nib: `Signed-off-by: <my@email.address>` row (via `git commit --signoff`)

### CODE REVIEW

File: docs/LuCI-0.10.md [hunk 1]
```diff
@@ -192,5 +192,5 @@ Each theme must include this file in the `<head>` area of the document for forms
 It should be included like this:
 
 ```html
-<script type="text/javascript" src="<%=resource%>/xhr.js"></script>
+<script src="<%=resource%>/xhr.js"></script>
```
[the remark this answers is not in the document]
Comment (Rupurudu) [reply docs/LuCI-0.10.md:195]: I updated the github wiki. Thanks.

[the remark this answers is not in the document]
Comment (Rupurudu) [reply docs/LuCI-0.10.md:195]: I was waiting for @systemcrash to comment on it but I'll squashed it anyway.

**Comment by systemcrash** [comment]
Don't touch the br tags.

**Comment by Rupurudu** [comment]
Okay, I'll keep the old XHTML style void tag ending.

**Comment by systemcrash** [comment]
Unless you've tested every app with its changes, please leave those CDATA blocks. I'm as yet unsure of the effect of their absence.

**Comment by systemcrash** [comment]
If someone is still enforcing X(HT)ML browsing somewhere, they're needed, I think. Can you test @stokito ?

**Comment by Rupurudu** [comment]
> If someone is still enforcing X(HT)ML browsing somewhere, they're needed, I think. Can you test @stokito ?

In order to use XHTML mode, you'll have to send content with content type `application/xhtml+xml` otherwise, the browser engine uses the HTML mode. We always send `text/html` so there is no way that any standards compliant parser use XHTML parsing on the content anyway.

https://developer.mozilla.org/en-US/docs/Glossary/XHTML

Like @stokito said, the only case this will break anything is someone's homemade custom parser that incorrectly uses XML parser mode, even through the content is HTML, not XHTML.

**Comment by systemcrash** [comment]
Merged. Let's see...

**Pull request merged**